interfaces/Portalicious/src/app/pages/project/project-team/project-team.component.html

```diff
@@ -3,5 +3,17 @@
   i18n-pageTitle="@@page-title-project-team"
   [projectId]="projectId()"
 >
-  <p i18n>project-team works!</p>
+  <div header-actions>
+    @if (canManageAidworkers()) {
+      <app-add-user-button />
+    }
+  </div>
+  <app-query-table
+    [columns]="columns()"
+    [query]="projectUsers"
+    localStorageKey="project-team-table"
+    [contextMenuItems]="contextMenuItems()"
+    (onUpdateContextMenuItem)="selectedUser.set($event)"
+    [globalFilterFields]="['username', 'roleString', 'scope']"
```

Comment:
Is there a way to 'connect' this more?
So not using "just strings", but using actual references/enums/etc?

Reply:
It is currently defined in the [query-table component](https://github.com/global-121/121-platform/pull/5693/files#diff-1907126206c4fe9833dff805793412afcfc7a57c079e5ea32e58150178ff7d71R69) as

`globalFilterFields = input<(keyof TData & string)[]>();`

Which means TS will throw an error if you try to write a string that isn't a property of TData. TData is typed dynamically as the return type of the mutation.

Reply:
Aha, so its sort-of a "one way connection you cannot violate from the other side". 👍